src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -518,6 +518,7 @@ private static bool IsValidDefaultValue(BoundExpression expression)
                 switch (expression.Kind)
                 {
                     case BoundKind.DefaultExpression:
```

Comment:
>BoundKind.DefaultExpression [](start = 25, length = 27)

Is this case reachable?

Reply:
Yes, the check for default value occurs before conversions.

---
In reply to: [314122197](https://github.com/dotnet/roslyn/pull/37596#discussion_r314122197) [](ancestors = 314122197)